src/components/CreateNewEvent.js

```diff
@@ -312,54 +316,75 @@ function CreateNewEvent() {
                         shrink: true
                       }}
                       style={{ marginRight: '30px' }}
+                      validators={['required']}
+                      errorMessages={['This is a required field']}
                     />
                   </Grid>
                   <Grid item xs={12} sm={12} md={6}>
-                    <TextField
+                    <TextValidator
                       fullWidth
                       className={classes.input1}
                       type="time"
                       name="time"
                       value={formData.time}
                       id="time"
                       defaultValue="07:30"
-                      variant="outlined"
+                      // variant="outlined"
                       onChange={handleChange}
+                      validators={['required']}
+                      errorMessages={['This is a required field']}
                     />
                   </Grid>
                 </Grid>
 
-                <TextField
+                <TextValidator
                   fullWidth
                   className={classes.textField}
                   placeholder="Event link / Registration link"
                   name="eventLink"
                   value={formData.eventLink}
-                  variant="outlined"
+                  // variant="outlined"
                   onChange={handleChange}
+                  validators={[
+                    'required']}
+                  errorMessages={[
+                    'This is a required field',
+                    'Please enter a valid contact number'
+                  ]}
                 />
                 {speaker.map((item, idx) => (
-                  <fieldset className={classes.socialLinks}>
-                    <InputBase
-                      className={classes.social}
+                  <fieldset className={classes.speakerInput}>
+                    <TextValidator
+                      className={classes.speaker}
                       fullWidth
                       id={idx.toString()}
                       name="speakerName"
-                      value={speaker.speakerName}
+                      value={item.speakerName}
                       placeholder="Speaker Name"
                       onChange={handleSpeakerChange}
+                      validators={['required']}
+                      errorMessages={['This is a required field']}
                     />
-                    <InputBase
-                      className={classes.social}
+                    <TextValidator
+                     key="linkedIn"
```

Comment:
indentation error use preferred code formator to resolve all indent-related errors in one go.

Reply:
I already formatted the code with prettier but ok I am doing changes.